### ISSUE TITLE
Tables exported in duplicate

### ISSUE THREAD

**Issue by alexsykes** [Sequel-Ace/Sequel-Ace#1997]
- Sequel Ace Version (including build number): 4.0.15
- Sequel Ace Source (App Store/GitHub/Homebrew): AppStore
- macOS Version: 14.4.1
- Processor Type (Intel/Apple): Apple
- MySQL Version: 8.0.34
- macOS Localization: en/gb

**Is Issue Present in [Latest Beta](https://github.com/Sequel-Ace/Sequel-Ace/releases)?**

**Description**
Table SQL dump on File/Export… exports table data and structure twice. 

**Steps To Reproduce**
1. Select database, tables to export
2. Right-click on table(s), Export/as SQL dump…
3. If one table selected, dialog shows '1 of 2, 2 of 2' being exported.
4. On reading the dump file, same table exported twice. 
5. Similar occurs when mutliple tables selected for dump

**Expected Behaviour**
Table content and data dumped only once.

**Related Issues**

**Additional Context**

**Comment by Jason-Morcos** [comment]
Do the tables exported in duplicate happen to be "favorited" tables, or is this happening for all tables?

**Comment by alexsykes** [comment]
Yes, I believe they are. Thanks,AlexSent from my iPhone 13On 20 Apr 2024, at 17:06, Jason Morcos ***@***.***> wrote:﻿
Do the tables exported in duplicate happen to be "favorited" tables, or is this happening for all tables?

**Comment by Jason-Morcos** [comment]
Got it - that's a duplicate of this other issue then! https://github.com/Sequel-Ace/Sequel-Ace/issues/1501
PRs are super welcome if someone wants to take a stab at addressing this one
